superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -126,16 +126,58 @@ def add_parser_arguments(self):
                     help='Number of threads in %s test.' % io_str,
                 )
 
-    def _preprocess(self):
+        self._parser.add_argument(
+            '--numa_node',
+            type=int,
+            required=False,
+            help='The index of NUMA node.',
+        )
+        self._parser.add_argument(
+            '--verify',
+            type=str,
```

Comment:
shall we add some choices?

Reply:
I expect this a pure wrapper of fio's `--verify`: https://fio.readthedocs.io/en/latest/fio_doc.html#cmdoption-arg-verify. It should abort with error when not meeting fio's expectation.

Reply:
can we add this link into 'help'? it can help user know which values should be set